code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -50,13 +50,24 @@
 /datum/chemical_reaction/plasmasolidification
 	name = "Solid Plasma"
 	id = "solidplasma"
-	required_reagents = list("iron" = 5, "frostoil" = 5, "plasma" = 20)
+	required_reagents = list("frostoil" = 5, "plasma" = 20)
```

Comment:
@nullbear yes they can be triggered inside mob afaik (it has mob_react = 1)
And just because it's still possible for a person to do the trick doesn't mean which should make it easy.